client/app/l10n/en.json

```diff
@@ -550,18 +550,18 @@
   "Security awareness panel title": "Security awareness panel title",
   "Select all": "Select all",
   "Select all recipients by default": "Select all recipients by default",
-  "Select an option:": "Select an option:",
+  "Select an option:": "Select something:",
   "Select the context of your submission:": "Select the context of your submission:",
   "Select the question to be triggered when this option is selected": "Select the question to be triggered when this option is selected",
-  "Select the step to be triggered when this option is selected": "Select the step to be triggered when this option is selected",
+  "Select the step to be triggered when this option is selected": "Select an automatic follow-up reaction for this option",
   "Select this recipient.": "Select this recipient.",
   "Selection box": "Selection box",
   "Send": "Send",
   "Send a test email to your email address.": "Send a test email to your email address.",
   "Sending submissions over HTTPS exposes the act of doing so, but not the content.\\n\\nAlso, hiding the fact that you are submitting something makes the submission anonymous, which is strongly recommended.\\n\\nThis can be achieved with the use of the Tor Browser, providing anonymisation not found in other browsers like Internet Explorer, Firefox, Chrome, Opera etc.": "Sending submissions over HTTPS exposes the act of doing so, but not the content.\n\nAlso, hiding the fact that you are submitting something makes the submission anonymous, which is strongly recommended.\n\nThis can be achieved with the use of the Tor Browser, providing anonymisation not found in other browsers like Internet Explorer, Firefox, Chrome, Opera etc.",
   "Server temporarly unusable due to resource exhaustion; advise the Admin if you can.": "Server temporarly unusable due to resource exhaustion; advise the Admin if you can.",
   "Session expired": "Session expired",
-  "Set a PGP public key for encryption of email notifications": "Set a PGP public key for encryption of email notifications",
+  "Set a PGP public key for encryption of email notifications": "Upload the public part of a PGP-key to encrypt e-mail notifications with it",
```

Comment:
@comradekingu: going to change this to be simply: "Upload your public PGP key to enable encryption of submissions".

Reply:
ah no actually i'm removing this by reusing: "Set up encryption by providing a PGP public key"
this way we could reduce the application strings.

Reply:
They are called public PGP keys, not PGP public keys. This one time i actually think it is warranted to explain what is going on.